app-drawer/app-drawer.html

```diff
@@ -120,39 +145,182 @@
 
       is: 'app-drawer',
 
+      behaviors: [
+        Polymer.AppDocumentScrollBehavior
+      ],
+
       properties: {
 
         /**
-         * Specifies whether or not the drawer is opened.
+         * The opened state of the drawer.
          */
         opened: {
           type: Boolean,
           value: false,
           notify: true,
-          reflectToAttribute: true
+          reflectToAttribute: true,
+          observer: '_openedChanged'
         },
 
         /**
-         * Determinates the position of the drawer on the screen.
-         * Options are left, right, top, bottom.
+         * The drawer does not have a scrim and cannot be swiped close.
+         */
+        persistent: {
+          type: Boolean,
+          value: false,
+          reflectToAttribute: true,
```

Comment:
Yes, I was mindful of limiting usage of `reflectToAttribute`. In this particular case, it is used for styling.